x-pack/plugin/ml/build.gradle

```diff
@@ -80,7 +80,7 @@ artifacts {
 
 tasks.register("extractNativeLicenses", Copy) {
   dependsOn configurations.nativeBundle
-  into "${buildDir}"
+  into "${buildDir}/extractedNativeLicenses"
```

Comment:
use a dedicated output folder for extracted native licenses. This avoids not having the whole `buildDir` considered a task output here